templates/email/design.js

```diff
@@ -84,28 +95,24 @@ module.exports = cx.design
       .withGroupId('images-hRO9mw')
       .withLabel('Bilder')
       .withContentElements(
-        require('@bsi-cx/design-standard-library-email/content-elements/base/image'),
-        require('./content-elements/base/img-66'),
-        require('./content-elements/base/img-50'),
-        require('./content-elements/base/img-33'),
+        require('./content-elements/base/img-with-style').withMinVersion(Version.CX_23_2),
+        require('@bsi-cx/design-standard-library-email/content-elements/base/image').withArchivedMinVersion(Version.CX_23_2),
+        require('./content-elements/base/img-66').withArchivedMinVersion(Version.CX_23_2),
+        require('./content-elements/base/img-50').withArchivedMinVersion(Version.CX_23_2),
+        require('./content-elements/base/img-33').withArchivedMinVersion(Version.CX_23_2),
         require('@bsi-cx/design-standard-library-email/content-elements/base/img-10')),
-    cx.contentElementGroup
-      .withGroupId('buttons-BadRb3')
-      .withLabel('Buttons')
-      .withContentElements(
-        require('@bsi-cx/design-standard-library-email/content-elements/base/cta'),
-        require('./content-elements/base/cta-center'),
-        require('./content-elements/base/cta-right')),
     cx.contentElementGroup
       .withGroupId('special-content-5C921n')
-      .withLabel('Besonderer Inhalt')
+      .withLabel('Inhalt')
       .withContentElements(
+        require('@bsi-cx/design-standard-library-email/content-elements/base/cta'),
+        require('./content-elements/base/cta-center').withArchivedMinVersion(Version.CX_23_2),
+        require('./content-elements/base/cta-right').withArchivedMinVersion(Version.CX_23_2),
         require('@bsi-cx/design-standard-library-email/content-elements/base/highlighted-content')
           .withExtendedDropzone(
             'highlighted-content-dropzone-4HtYBB',
             ...contentElements,
             ...spacerElements),
-        require('@bsi-cx/design-standard-library-email/content-elements/base/news-snippet'),
```

Comment:
Did you intend to remove this? if yes, please explain in README

Reply:
This was unintentional and was therefore added again.